pkg/assetvalidator/validateAssets.go

```diff
@@ -50,6 +50,9 @@ func (v *NexposeAssetValidator) getScanTime(asset domain.Asset, scanID string) (
 				return scanTime, nil
 			}
 		}
+		if evt.Type == "AGENT-IMPORT" && time.Since(scanTime).Hours() < 24 {
```

Comment:
nit: These two `if` statements are mutually exclusive and so should probably be written as a case switch.

Reply:
good point. Pushing up a rework now.